packages/perseus/src/widgets/interactive-graphs/graphs/point.tsx

```diff
@@ -46,6 +46,16 @@ export function UnlimitedPointGraph(props: PointGraphProps) {
         height,
     ]);
     const [[left, top]] = useTransformVectorsToPixels([minX, maxY]);
+    const itemsRef = React.useRef<Array<SVGElement | null>>([]);
+
+    React.useEffect(() => {
```

Comment:
If the learner focuses something outside the graph, and then the graph re-renders, will this hook force focus back to the point? Or no, because we unset `graphState.focusedPointIndex` when the point blurs?

Reply:
In experimenting with this it'll cause the point to blur and that will unfocus the point